ui/src/views/WorkerManager/routes.js

```diff
@@ -1,16 +1,16 @@
 import lazy from '../../utils/lazy';
 
-const WorkerManagerViewWorkerTypes = lazy(() =>
-  import(/* webpackChunkName: 'WorkerManager.WMViewWorkerTypes' */ './WMViewWorkerTypes')
+const WorkerManagerViewWorkerPools = lazy(() =>
+  import(/* webpackChunkName: 'WorkerManager.WMViewWorkerPools' */ './WMViewWorkerPools')
 );
 const WorkerManagerViewWorkers = lazy(() =>
```

Comment:
Should this be `WorkerManagerViewWorkerPool`?

Reply:
No, line 6 is correct. Comment on line 7 is not!